heat/core/linalg/basics.py

```diff
@@ -827,6 +827,225 @@ def norm(a):
     return exponential.sqrt(d).item()
 
 
+def outer(a, b, out=None, split=None):
+    """
+    Compute the outer product of two 1-D DNDarrays.
+
+    Given two vectors, ``a`` = [a_0, a_1, ..., a_n] and ``b`` = [b_0, b_1, ..., b_m], the outer product is:
+
+    [[a_0*b_0  a_0*b_1 ... a_0*b_m]
+     [a_1*b_0     .
+     [ ...        .
+     [a_n*b_0     .        a_n*b_m]]
+
+    Parameters
+    ----------
+
+    a(n,): DNDarray
```

Comment:
Is this the way dimension information is displayed now with the new docstring format?

Reply:
fixed